### PR TITLE
feat: Add option to self-assign graduate role

### PR DESCRIPTION
**Pull request by xvanick1**
# Related issues
Resolves https://github.com/OqixDevs/OqixTS/issues/108

# Type of pull request
- [x] New feature
- [ ] Bug fix
- [ ] Refactoring
- [ ] Feature update
- [ ] Other

# Describe your PR in detail
This feature is adds an option for verified users to permanently self-assign role 'graduate'.

### CODE REVIEW

File: src/commands/graduate.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ */
+
+export const data = new SlashCommandBuilder()
+    .setName('graduate')
+    .setDescription('Identifies user permanently as graduate.')
+    .addStringOption((option) =>
+        option
+            .setName('confirmation')
+            .setDescription(
+                "Write 'confirm' if you want to set your status to graduate."
```
Comment (mimotej) [inline src/commands/graduate.ts:18]: Just a small nit - is this really necessary? Is this just supposed to make sure that you do not give yourself role accidentally or is there some other reason?

**Review by mimotej — COMMENTED** [review]
Just one small nit, otherwise
/lgtm

File: src/commands/graduate.ts (hunk 1 above)
Comment (xvanick1) [reply src/commands/graduate.ts:18]: Hi, it is exactly as you mentioned. Just an additional step to make sure that you do not give yourself role accidentally.

**Review by mimotej — APPROVED** [review]
/lgtm

**Pull request merged**